### ISSUE TITLE
Add a User Read-only partition to Data

### ISSUE THREAD

**Issue by curelom** [microsoft/appcenter#1231]
Would like to see an additional partition the would be read-only (similar to the DefaultPartitions.AppDocuments), but only visible to the specific user.

An example of why this could come in real handy:
We have a running account balance and we don't want the user to be able to modify this from the phone directly via hack, etc..  We only want it changed via APIs.  As this would be specific user data, we wouldn't want to put it in the more public AppDocuments as we don't want other users to have a chance of seeing this data.

**Comment by Oddj0b** [comment]
I've tagged my colleague who will take a look in the near future, thank you for spending your time providing feedback.

**Comment by Zakeelm** [comment]
@curelom  Thank you so much for your request. We announced this week in a [blog post]( https://devblogs.microsoft.com/appcenter/app-center-mbaas-retirement/) that we’re retiring the Auth and Data services later this year and recommending customers transition to Azure Active Directory B2C and Azure Cosmos DB instead. Let me know if you have any additional questions about this transition and i'll make sure to get them answered! Closing this issue.
